ais/api/views.py

```diff
@@ -304,8 +394,6 @@ def search_view(query):
     if len(query) + arg_len < 60:
 
         parsed = PassyunkParser().parse(query)
-        all_addresses = None
-        all_intersections = None
         search_type = parsed['type']
 
         if search_type == "intersection_addr":
```

Comment:
please refactor the intersection section into its own view, as though there an `/intersection` endpoint. `/search` should just call that view.